base/shell/explorer/lang/en-US.rc

```diff
@@ -196,3 +196,9 @@ BEGIN
     IDS_TASKBAR_STARTMENU_PROP_CAPTION "Taskbar and Start Menu"
     IDS_RESTORE_ALL "&Show Open Windows"
 END
+
+STRINGTABLE
+BEGIN
+    IDS_TRAYWND_UNDO_CASCADE "Undo &Cascade Windows"
+    IDS_TRAYWND_UNDO_TILE "Undo &Tile Windows"
```

Comment:
I have checked Windows 2003, only one of these menu items is visible at a time, so it uses the same mnemonic (&) key for all of these actions:

![undo_cascade](https://user-images.githubusercontent.com/578406/199720143-1db1d791-f97e-4d37-b61d-7ed6474a984f.png) ![undo_tile](https://user-images.githubusercontent.com/578406/199720161-d4f030c4-5f99-49fa-a38a-84119d15e3c5.png) ![undo_minimize](https://user-images.githubusercontent.com/578406/199720176-79c5b880-5547-42c9-88a4-7945963b5bdd.png)
```suggestion
    IDS_TRAYWND_UNDO_CASCADE "&Undo Cascade"
    IDS_TRAYWND_UNDO_TILE "&Undo Tile"
```